File: Array 101/Move_Zeros1.cpp
class Solution {
public:
    void moveZeroes(vector<int>& nums) {
        int last=0;
        for(int i=0;i<nums.size();i++)
        {
            if(nums[i]!=0)
                nums[last++]=nums[i];
        }
        for(int j=last;j<nums.size();j++)
            nums[j]=0;
        
    }
};

File: Array 101/Move_Zeros1.py
class Solution:
    def moveZeroes(self, nums: List[int]) -> None:
        """
        Do not return anything, modify nums in-place instead.
        """
        last=0
        i=0
        while(i<len(nums)):
            if nums[i]!=0:
                nums[last]=nums[i];
                last+=1
            i+=1
        for i in range(last,len(nums)):
            nums[i]=0;